ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivateStateKeyValueStorage.java

```diff
@@ -12,24 +12,34 @@
  */
 package org.hyperledger.besu.ethereum.privacy;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+
 import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.ethereum.core.Log;
+import org.hyperledger.besu.ethereum.core.LogSeries;
+import org.hyperledger.besu.ethereum.rlp.RLP;
 import org.hyperledger.besu.plugin.services.storage.KeyValueStorage;
 import org.hyperledger.besu.plugin.services.storage.KeyValueStorageTransaction;
 import org.hyperledger.besu.util.bytes.Bytes32;
 import org.hyperledger.besu.util.bytes.BytesValue;
+import org.hyperledger.besu.util.bytes.BytesValues;
 
+import java.util.List;
 import java.util.Optional;
 
 public class PrivateStateKeyValueStorage implements PrivateStateStorage {
 
+  private static final BytesValue EVENTS_KEY_SUFFIX = BytesValue.of("EVENTS".getBytes(UTF_8));
```

Comment:
Do you want to change that to TRANSACTION_LOGS_KEY_SUFFIX to make it consistent?